examples/package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "serve-example": "webpack serve --config webpack.dev.js",
+    "prebuild-example": "npm install",
```

Comment:
I forget, does this get run automatically from the top-level package.json, or do you have to explicitly do it?

Reply:
it gets called from the top level package.json by the GitHub action.